CHANGELOG.rst

```diff
@@ -47,6 +47,13 @@ Changelog
   a single-valued RDN.
 * Added
   :func:`~cryptography.hazmat.primitives.asymmetric.ec.derive_private_key`.
+* Added support to
+  :meth:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey.sign`
+  and
+  :meth:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey.verify`
+  RSA signatures with a
```

Comment:
When this gets parsed into HTML it will read "Added support to sign and verify RSA signatures with a Prehashed digest" where sign, verify, and Prehashed are all links. I'm happy to reword it if you'd like, but there aren't any missing words :)